app/models/bisect.py

```diff
@@ -116,50 +122,58 @@ def from_json(json_obj):
         return None
 
 
-class BootBisectDocument(BisectDocument):
+class CommonBisectDocument(BisectDocument):
```

Comment:
Yes I think they should be common to all bisections that can be run on kernelci.org: build, boot, tests...  The `BisectDocument` class only had generic attributes that would apply to any bisection, based entirely on source control revision.  So I wasn't sure if mixing things from builds and boots was a good idea, or why these fields were not already in `BisectDocument` in the first place.

Reply:
There were probably differences at the beginning, but I think all those fields can be in a single class since they definitely are common between the two.

Reply:
OK I've moved all the fields into the base class `BisectDocument` and updated unit tests accordingly.